libs/micronaut-snitch/src/main/java/com/agorapulse/micronaut/snitch/SnitchInterceptor.java

```diff
@@ -20,8 +20,10 @@
 import io.micronaut.aop.MethodInterceptor;
 import io.micronaut.aop.MethodInvocationContext;
 import io.micronaut.context.ApplicationContext;
+import io.micronaut.context.BeanContext;
 import io.micronaut.inject.qualifiers.Qualifiers;
 
+import javax.inject.Inject;
```

Comment:
@sdelamo here there is a new import which looks unused

Reply:
That it is a mistake.